plugins/icons_papirus/oomox_plugin.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+from oomox_gui.config import FALLBACK_COLOR
+from oomox_gui.export_common import FileBasedExportDialog
+from oomox_gui.plugin_api import OomoxIconsPlugin
+from oomox_gui.i18n import _
+
+
+PLUGIN_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+class PapirusIconsExportDialog(FileBasedExportDialog):
+    timeout = 100
+
+    def do_export(self):
+        self.command = [
+            "bash",
+            os.path.join(PLUGIN_DIR, "change_color.sh"),
+            "-o", self.theme_name,
+            self.temp_theme_path,
+        ]
+        super().do_export()
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.do_export()
+
+
+class Plugin(OomoxIconsPlugin):
+    name = 'papirus_icons'
+    display_name = 'Papirus'
+    export_dialog = PapirusIconsExportDialog
+    preview_svg_dir = os.path.join(PLUGIN_DIR, "icon_previews/")
+
+    theme_model_icons = [
+        {
+            'key': 'ICONS_THEME',
```

Comment:
please prefix it, like PAPIRUS_ICONTHEME or so

everything else looks fine, thanks a lot!